lib/cocoapods-core/cdn_source.rb

```diff
@@ -144,6 +160,10 @@ def versions(name)
     def specification_path(name, version)
       raise ArgumentError, 'No name' unless name
       raise ArgumentError, 'No version' unless version
+      unless versions(name).include?(Version.new(version))
+        raise StandardError, "Unable to find the specification #{name} " \
```

Comment:
nit of the nits wrap in backticks everything that is presented. Is `StandardError` or `Informative` here correct?

Reply:
I actually copied it from `Pod::Source` word-for-word. I'd rather keep it the same in both. Do you want to change it there too?

Reply:
ah oh well keep it as is!

Reply:
Cool!